docs/docs/Get-Started/get-started-installation.md

```diff
@@ -128,6 +128,22 @@ This error can occur during Langflow upgrades when the new version can't overrid
 
 If you wish to retain your files, back them up before clearing the folder.
 
+### Langflow installation freezes at pip dependency resolution
 
+Installing Langflow with `pip install langflow` slowly fails with this error message:
 
+```plain
+pip is looking at multiple versions of <<library>> to determine which version is compatible with other requirements. This could take a while.
+```
+
+To fix this issue, install Langflow with [uv](https://docs.astral.sh/uv/getting-started/installation/) instead of pip.
 
+```plain
+uv pip install langflow
```

Comment:
```suggestion
uv add langflow
```

[`uv add` command](https://docs.astral.sh/uv/reference/cli/#uv-add) is even better option here

Reply:
You can ignore the above suggestion.

Reply:
There's a good write up [here](https://github.com/astral-sh/uv/issues/9219)